plugin.audio.intergalacticfm/resources/streams.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
Are these URL's publicly available? If so, where?

Reply:
Yes at https://www.intergalactic.fm/ and under the header RADIO are three buttons. Each lead to a page that lists above the inline audio player ALTERNATIVE TUNE-IN and after that these URLs are listed. Does that answer the question?

Reply:
Yes at https://www.intergalactic.fm/ and under the header RADIO are three buttons. Each lead to a page that lists above the inline audio player ALTERNATIVE TUNE-IN and after that these URLs are listed. Does that answer the question? Does this need to go into the README?

Reply:
resolved in latest push